tasks/Resources/Provisioning/Hive/gpu-provision.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Create GPU Node In Self Managed AWS Cluster
+    Set Log Level    Info
```

Comment:
could you register the keyword as "protected" in the RHOSi setup instead of using Set Log Level?